### PR TITLE
Animals table

### PR DESCRIPTION
**Pull request by HunterX-7**
## Dear Code reviewer In this Project, I:

- Add the following features to the project:

### Vet Clinic project requirements

- Use [this template](https://github.com/microverseinc/curriculum-template-databases) to generate your repo.
- Create a database named `vet_clinic`.
- Create a table named `animals` with the following columns:
  - `id`: integer
  - `name`: string
  - `date_of_birth`: date
  - `escape_attempts`: integer
  - `neutered`: boolean
  - `weight_kg`: decimal
- Copy the SQL you used in the previous point into the `schema.sql` file in the generated repository.
- Insert the following data:
  - Animal: His name is Agumon. He was born on Feb 3rd, 2020, and currently weighs 10.23kg. He was neutered and he has never tried to escape.
  - Animal: Her name is Gabumon. She was born on Nov 15th, 2018, and currently weighs 8kg. She is neutered and she has tried to escape 2 times.
  - Animal: His name is Pikachu. He was born on Jan 7th, 2021, and currently weighs 15.04kg. He was not neutered and he tried to escape once.
  - Animal: Her name is Devimon. She was born on May 12th, 2017, and currently weighs 11kg. She is neutered and she has tried to escape 5 times.
- Copy the SQL you used in the previous point into the `data.sql` file in the generated repository.

   CreateTable

- Write queries for the following quests:
  - Find all animals whose name ends in "mon".
    AllMonEnds
  - List the name of all animals born between 2016 and 2019. 
    dateOfBirth2016-2012019
  - List the name of all animals that are neutered and have less than 3 escape attempts.
    NeuteredAndEscapedattempts3
  - List the date of birth of all animals named either "Agumon" or "Pikachu".
    AgumonOrPikachu
  - List name and escape attempts of animals that weigh more than 10.5kg
    weightAndEscapeattempts
  - Find all animals that are neutered.
    whereNeuteredTrue
  - Find all animals not named Gabumon.  
    NotGabumon
  - Find all animals with a weight between 10.4kg and 17.3kg (including the animals with weights that equals precisely 10.4kg or 17.3kg)
    weigthbetween10 4and17 3
- **Take a screenshot of the results of your queries.**
- Copy the SQL you used in the previous point into the `queries.sql` file in the generated repository.
- Remember to add the screenshot with the results of your queries to your Pull Request description.

## I Hope you have a great day and your support is much appreciated! 😄

### CODE REVIEW

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @HunterX-7  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights :heavy_check_mark: 
- [x] Gitflow is used correctly :white_check_mark: 
- [x] Good descriptive commit messages :white_check_mark:  
- [x] Descriptive pull  request title and summary :white_check_mark: 

## Required Changes ♻️
- [ Required ] I've noticed that the License link on your readme file does not lead to the License in the project root directory, please consider changing the License Link to the one below to fix this. A working License link informs someone who would like to use your code or project how they are Licensed.

```
This project is [LICENSE](./LICENSE) licensed.
```
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by HunterX-7** [comment]
Thanks for the suggestion the branch am currently asking for a review has the change in the readme file with the correct license link. 😄

File: README.md [hunk 1]
```diff
@@ -240,52 +137,30 @@ Example:
...
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
-
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+This project is [MIT](https://choosealicense.com/licenses/mit/) licensed.
```
Comment (Gambit142) [inline README.md]: - [ ] The first reviewer requested that your current **_MIT LICENSE_** link to the correct _**LICENSE**_ file in the root directory. Currently, when I click the _**LICENSE**_ link, I am redirected to the general _**MIT**_ template site. This link in your _**README.md**_ file should direct me to the _**LICENSE**_ in the root directory. Kindly implement this change. 👍 💻

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @HunterX-7 👋, 

Great work on making the changes requested by a previous reviewer 👏 

## Highlights
- Everything that creates or changes the structure of the tables is in the _**schema.sql**_ file
- Everything that populates tables with data is in the _**data.sql**_ file
- Everything that is a query that helps to analyze data is in the _**queries.sql**_ file

You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

 ## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please, remember to tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Felix45** [comment]
> Thanks for the suggestion the branch am currently asking for a review has the change in the readme file with the correct license link. smile

Hi, @HunterX-7 when I click the License link on your readme file it redirects me to the `MIT` LIcense template, this link should lead to the `License` file in the root directory of your project. Please check on this, currently, it opens this Link [https://choosealicense.com/licenses/mit/](https://choosealicense.com/licenses/mit/) which should not be the case.

**Comment by HunterX-7** [comment]
Thank you I did not understand that correctly really appreciated the feedback and your extra time to clarify! 😄

**Review by danmainah — APPROVED** [review]
Hi @HunterX-7 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ Gitflow was correctly used
✔️  Good PR and Description

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**